client/src/app/components/CreateProject/Options/Option.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import "./options.css";
+const classNames = require("classnames");
+
+class Option extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { isActive: this.props.isActive };
+  }
+
+  componentDidMount() {
+    if (this.props.isDefault === this.props.name) {
```

Comment:
Even here you would benefit from the destructuring, since you have variables that you're accessing with this.props.sth. 

As a general rule, as soon as you have more than one occurrence of this.props.sth, then it's good to destructure them on the top of the method to avoid repetition, and for a cleaner code